src/install/NetworkTask.zig

```diff
@@ -70,6 +72,35 @@ fn countAuth(header_builder: *HeaderBuilder, scope: *const Npm.Registry.Scope) v
     header_builder.count("npm-auth-type", "legacy");
 }
 
+// if this package is likely to have a "libc" field in its package.json.
+// npm's optimised response doesn't include it, so we need to use the unoptimised one
+fn isPlatformSpecificPackage(name: string) bool {
+    if (name.len < 5) return false;
+    const platform_keywords = [_][]const u8{
+        "musl", "linux", "x64", "aarch64", "glibc", "arm64",
+    };
+
+    if (name[0] == '@') {
+        if (strings.indexOfChar(name, '/')) |i| {
+            const scope_and_pkg = name[i + 1 ..];
+            inline for (platform_keywords) |keyword| {
+                if (strings.containsComptime(scope_and_pkg, keyword)) {
+                    return true;
+                }
+            }
+        }
+    } else if (strings.indexOfChar(name, '-')) |i| {
```

Comment:
```suggestion
    if (strings.indexOfChar(name, '/')) |i| {
        const scope_and_pkg = name[i + 1 ..];
        inline for (platform_keywords) |keyword| {
            if (strings.containsComptime(scope_and_pkg, keyword)) {
                return true;
            }
        }
    } else if (strings.indexOfChar(name, '-')) |i| {
```

just makes it a little more simple. we can assume the package name is valid. also, when the package has a '/', should we also check for '-' before looking for keywords?

Reply:
I was half trying to have best perf as who knows how many times this fn gets ran (yes i know its not too much, but every little thing right)

Yeah we *could* check for `-` there too but im not sure if its going to gain much and I haven't checked enough  to see whats in use (ie is there a @mybuild/win32 or smth)

Reply:
`@mybuild/win32` is a good example, wouldn't be surprised if this is common. let's not check for '/' if it's scoped